test/offline/storage_unit.js

```diff
@@ -869,6 +869,43 @@ describe('Storage', function() {
         }).catch(fail).then(done);
       });
     });  // describe('default track selection callback')
+
+    describe('temporary license', function(){
+
+      beforeEach(function(){
+        storage.configure({ isPersistentLicense: false });
```

Comment:
This doesn't configure DRM, so the tests below are all exercising clear content paths.  See the `drmEngine` setup in the `'stores DRM info'` test and do the same here in `beforeEach`.